lib/mongoose_push.ex

```diff
@@ -1,18 +1,52 @@
 defmodule MongoosePush do
   @moduledoc """
-  Documentation for MongoosePush.
+  MongoosePush is simple (seriously) service providing ability to send push
+  notification to `FCM` (Firebase Cloud Messaging) and/or
+  `APNS` (Apple Push Notification Service). What makes it cool is not only
+  simplicity but also support for newest and fastest `HTTP/2` based APIs
+  for both services.
+
+  At this moment only those two services are supported but in future
+  MongoosePush may and probably will support even more Push Notification Services.
   """
 
-  @doc """
-  Hello world.
+  require Logger
+  alias MongoosePush.Application
```

Comment:
I don't like this alias. It's misleading as at first glance you don't know if it refers to `Elixir.Application` or to `Elixir.MngoosePush.Application`.